frontend/src/app/modules/ifc_models/bcf/container/bcf-container.component.ts

```diff
@@ -1,47 +1,67 @@
-import {Component} from "@angular/core";
+import {ChangeDetectorRef, Component, Injector, OnInit} from "@angular/core";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {PathHelperService} from "core-app/modules/common/path-helper/path-helper.service";
 import {CurrentProjectService} from "core-components/projects/current-project.service";
 import {WorkPackageTableConfigurationObject} from "core-components/wp-table/wp-table-configuration";
-import { StateService } from '@uirouter/core';
+import {StateService} from '@uirouter/core';
 import {GonService} from "core-app/modules/common/gon/gon.service";
+import {QueryParamListenerService} from "core-components/wp-query/query-param-listener.service";
+import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
+import {WorkPackagesListService} from "core-components/wp-list/wp-list.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/2bca7ebe6a58776d5479934ae44cca26a6cab30c/frontend/src/app/modules/ifc_models/bcf/container/bcf-container.component.ts#L11"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>